src/main/scala/tile/RocketTile.scala

```diff
@@ -159,6 +105,9 @@ class RocketTile private(
     if (!rocketParams.boundaryBuffers) super.makeSlaveBoundaryBuffers
     else TLBuffer(BufferParams.flow, BufferParams.none, BufferParams.none, BufferParams.none, BufferParams.none)
   }
+
+  val rocketLogicalTree: RocketLogicalTreeNode = new RocketLogicalTreeNode(cpuDevice, rocketParams, dtim_adapter, p(XLen), iCacheLogicalTreeNode)
+  //LogicalModuleTree.add(rocketLogicalTree, iCacheLogicalTreeNode)
```

Comment:
I think the commented out way is the ideal way of doing it; then it could be added to the tree at the point where the `ICache` is added, in `HasICacheFrontend`